Document Compass Provisioner component (#497)
kyma-incubator/compass: docs/provisioner/01-01-provisioner-overview.md

---
title: Overview
type: Overview
---

The Runtime Provisioner is a Compass component responsible for provisioning, installing, and deprovisioning clusters with Kyma (Kyma Runtimes). 

> **NOTE:** Kyma installation is not implemented yet. 

It is powered by [Hydroform](https://github.com/kyma-incubator/hydroform) and it allows you to provision the clusters in two ways:
- [directly on Google Cloud Platform (GCP)](08-01-provisioning-gcp.md)
- [through Gardener](08-02-provisioning-gardener.md) on:
    * Google Cloud Platform (GCP)
    * Microsoft Azure
    * Amazon Web Services (AWS).
    
Note that the operations of provisioning and deprovisioning are asynchronous. They return the operation ID, which you can use to [check the Runtime Operation Status](08-03-runtime-operation-status.md).

> The Runtime Provisioner also allows you to [clean up Runtime data](08-06-clean-up-runtime-data.md). This operation removes all the data concerning a given Runtime from the database and frees up the Runtime ID for reuse. It is useful when your cluster has died or when the operation of deprovisioning has failed.

Reviewer: ```suggestion
The Runtime Provisioner also allows you to [clean up Runtime data](08-06-clean-up-runtime-data.md). This operation removes all the data on a given Runtime from the database and frees up the Runtime ID for reuse. It is useful when your cluster goes down or when the operation of deprovisioning fails.
```

Reviewer: `on` or `about`

Author: It doesn't just wipe the data on the Runtime, it removes the Runtime from the database altogether. 
I'll try to rephrase it.

Follow the links to access the respective tutorials. 

>     

Reviewer: which links do you mean here? If those above, I think that sentence is redundant or you could give an explicit list of tutorials somehow descriptively here, with bullet points?
